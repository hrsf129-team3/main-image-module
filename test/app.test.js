import React from 'react';
import App from '../client/src/components/app.jsx';

describe('App', () => {
  it('should render app component', () => {
    const component = shallow(<App />);
    expect(component).toMatchSnapshot();
  });
})

// describe('App', () => {
//   const baseURL = "http://localhost:3007";
//   const status = [{
//     ...
//   }];
//   before(() => {
//     fetchMock.get(`${baseURL}/api`, status);
//   });

//   after(() => {
//     fetchMock.reset();
//     fetchMock.restore();
//   });

//   it('contains the correct number of components', (done) => {
//     const wrapper = mount(<ParentComponent />);

//     //This is not working - the length of ChildComponent is always 0
//     expect(wrapper.find(ChildComponent).length).to.equal(status.length);
//   });
// });

// it('contains the correct number of components', (done) => {
//   const wrapper = mount(<App />);
//   // wait for the API call has benn resolved
//   setTimeout() => {
//       wrapper.update();
//       expect(wrapper.find(ChildComponent).length).to.equal(status.length);
//       done();
//   });
// });