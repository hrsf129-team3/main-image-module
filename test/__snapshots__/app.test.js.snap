// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render app component 1`] = `ShallowWrapper {}`;

exports[`Image should render image component 1`] = `ShallowWrapper {}`;
